[2022-02-21 16:53:28,791] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-09T00:00:00+00:00 [queued]>
[2022-02-21 16:53:28,806] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-09T00:00:00+00:00 [queued]>
[2022-02-21 16:53:28,806] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:28,806] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 16:53:28,806] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:28,820] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-02-09 00:00:00+00:00
[2022-02-21 16:53:28,825] {standard_task_runner.py:52} INFO - Started process 138 to run task
[2022-02-21 16:53:28,828] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-02-09T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpuw4u4rjb', '--error-file', '/tmp/tmp1fv20fr7']
[2022-02-21 16:53:28,829] {standard_task_runner.py:77} INFO - Job 43: Subtask get_path
[2022-02-21 16:53:28,882] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-02-09T00:00:00+00:00 [running]> on host 7c24e16e6467
[2022-02-21 16:53:28,939] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-02-09T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-09T00:00:00+00:00
[2022-02-21 16:53:28,940] {logging_mixin.py:109} INFO - /opt/***
[2022-02-21 16:53:28,940] {python.py:175} INFO - Done. Returned value was: None
[2022-02-21 16:53:28,952] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220209T000000, start_date=20220221T165328, end_date=20220221T165328
[2022-02-21 16:53:29,000] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 16:53:29,068] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 17:02:11,319] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-09T00:00:00+00:00 [queued]>
[2022-02-21 17:02:11,334] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-09T00:00:00+00:00 [queued]>
[2022-02-21 17:02:11,334] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:02:11,334] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 17:02:11,335] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:02:11,348] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-02-09 00:00:00+00:00
[2022-02-21 17:02:11,354] {standard_task_runner.py:52} INFO - Started process 136 to run task
[2022-02-21 17:02:11,357] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-02-09T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpt4_eo7oc', '--error-file', '/tmp/tmpe55owne7']
[2022-02-21 17:02:11,357] {standard_task_runner.py:77} INFO - Job 41: Subtask get_path
[2022-02-21 17:02:11,413] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-02-09T00:00:00+00:00 [running]> on host ce809d68944a
[2022-02-21 17:02:11,465] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-02-09T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-09T00:00:00+00:00
[2022-02-21 17:02:11,466] {python.py:175} INFO - Done. Returned value was: /nba_database
[2022-02-21 17:02:11,483] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220209T000000, start_date=20220221T170211, end_date=20220221T170211
[2022-02-21 17:02:11,529] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 17:02:11,579] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 19:10:54,421] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-09T00:00:00+00:00 [queued]>
[2022-02-21 19:10:54,429] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-09T00:00:00+00:00 [queued]>
[2022-02-21 19:10:54,429] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:54,429] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 19:10:54,429] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:54,438] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-02-09 00:00:00+00:00
[2022-02-21 19:10:54,442] {standard_task_runner.py:52} INFO - Started process 136 to run task
[2022-02-21 19:10:54,444] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-02-09T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp5kssqk9f', '--error-file', '/tmp/tmpc8waih4b']
[2022-02-21 19:10:54,445] {standard_task_runner.py:77} INFO - Job 41: Subtask get_path
[2022-02-21 19:10:54,489] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-02-09T00:00:00+00:00 [running]> on host 1594888d6c38
[2022-02-21 19:10:54,552] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-02-09T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-09T00:00:00+00:00
[2022-02-21 19:10:54,554] {python.py:175} INFO - Done. Returned value was: ['/home/***/.local/bin', '/usr/local/lib/python37.zip', '/usr/local/lib/python3.7', '/usr/local/lib/python3.7/lib-dynload', '/home/***/.local/lib/python3.7/site-packages', '/usr/local/lib/python3.7/site-packages', '/opt/***/dags', '/opt/***/config', '/opt/***/plugins']
[2022-02-21 19:10:54,593] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220209T000000, start_date=20220221T191054, end_date=20220221T191054
[2022-02-21 19:10:54,618] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 19:10:54,648] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
